dependencywalker.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = Dependency Walker
+Description = A free utility that scans any 32-bit or 64-bit Windows module (exe, dll, ocx, sys, etc.) and builds a hierarchical tree diagram of all dependent modules.
```

Comment:
```suggestion
Description = MsDepends is a free utility that scans any 32-bit or 64-bit Windows module (exe, dll, ocx, sys, etc.) and builds a hierarchical tree diagram of all dependent modules.
```
I want to add the term "MSDepends" here, because people might enter that into the rapps searchbox! So it will be found then also.